### ISSUE TITLE
signed releases

### ISSUE THREAD

**Issue by shibumi** [vivien/i3blocks#230]
Hello,
I am the maintainer of your software in Arch Linux. I would like to suggest to sign the releases. You can find a tutorial for it here: https://wiki.debian.org/Creating%20signed%20GitHub%20releases

Thanks

**Comment by vivien** [comment]
Thanks for the link and for maintaining an i3blocks package! The next release (coming soon) will definitely be signed. I will keep you guys updated.

**Comment by dikiaap** [comment]
Still waiting for this one! May I ask when the next release is released?

**Comment by shibumi** [comment]
Hi @vivien could you please provide a new signed release? People in Arch Linux start complaining about your software since months, because it behaves weird. I ship version 1.4 at the moment.. still waiting for your new release.

**Comment by vivien** [comment]
Hi @shibumi, I've bumped the version in the source to 1.5, but I did not tag a release yet. I wanted to sync with you (and others maintainers like @jpleau) to make sure you guys are aware that this i3blocks repository does not contain default scripts anymore. It only contains the core scheduler (i.e. the i3blocks binary). The default scripts have been moved to the [i3blocks-contrib](https://github.com/vivien/i3blocks-contrib) repository, maintained by @kb100 mainly.

At this point, I imagine the distros maintainers either packaging i3blocks with some scripts picked up from the contrib repository (or even some distro specific scripts!), or packaging i3blocks-contrib as well as a dependency (hard or soft) to i3blocks.

Regardless the approach, what else do you need from me except a signed tag in the i3blocks repository?

**Comment by shibumi** [comment]
@vivien I would prefer a signed tarball, but a signed tag works as well..

regarding i3blocks-contrib. I will most likely package it separately or someone will package it in the AUR.

**Comment by vivien** [comment]
@shibumi my only concern is that a fresh install of the i3blocks package on a new system would result in an empty bar. We must at least provide a minimalist system-wide config file to the user, something like the following:

```` ini
[notice]
full_text=Message to user here
color=#FFA500

[time]
command=date '+%Y-%m-%d %H:%M:%S'
interval=1
````

Do you think that this is the responsibility of the distro maintainer to include one in the i3blocks package?

**Comment by vivien** [comment]
@shibumi I've added a minimal system-wide configuration file that is installed by default with the package. Do what you want with it, keep it as is or override it is default scripts or Arch specific ones. It isn't meant to bring dependencies to the package though, as users are supposed to provide their blocks and configuration file.

I'm closing this issue since I've released a [verified 1.5 version](https://github.com/vivien/i3blocks/releases/tag/1.5) and will do so from now on.
